keras_hub/src/models/segformer/segformer_backbone_presets.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 The KerasHub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""SegFormerBackbone model preset configurations."""
+
+presets_no_weights = {
+    "segformer_b0_backbone": {
```

Comment:
preset names follow this convention `<architecture_name>_<dataset_name>`

Reply:
This'll depend on the weights we end up porting. Will update when we decide on which weights to port specifically

Reply:
Would it make sense to begin with random weights (just the arch):

- `from_preset(<architecture_name>)`

And then add new weights from specific datasets in a separate PR:

- `from_preset(<architecture_name>_<dataset_name1>)`
- `from_preset(<architecture_name>_<dataset_name2>)`

Primarily comes from the fact that we'll have to first do this for MiTs and this'll have to hang until then.